python/ray/dashboard/client/README.md

```diff
@@ -0,0 +1,44 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Let's clean up this readme

Reply:
Add how the UI is composed, like the react components hierarchy.
And gif showcasing it.